code/controllers/configuration.dm

```diff
@@ -78,10 +78,10 @@
 
 	var/server
 	var/banappeals
-	var/wikiurl = "http://baystation12.net/wiki"
-	var/forumurl = "http://baystation12.net/forums/"
+	var/wikiurl = "http://example.org"
+	var/forumurl = "http://example.org"
 
-	var/media_base_url = "http://nanotrasen.se/media" // http://ss13.nexisonline.net/media
+	var/media_base_url = "http://example.org"

```

Comment:
Yea - I'll add it to the example file. That should be clear enough (right below the forum/wiki link).